app/views/layouts/application.html.erb

```diff
@@ -24,8 +24,11 @@
       <% end %>
       <% flash.each do |type, message| %>
         <% if message.present? && message.is_a?(String) %>
-          <div><%= notice %></div>
-          <div><%= alert %></div>
+          <div class="flex flex-col items-center justify-center fixed bottom-[64px] w-full z-20">
+            <div data-controller="closable" class="bg-gray-3 rounded-lg px-5 py-4 absolute transition-all bottom-0 max-w-screen-sm text-white z-20 sticky">
```

Comment:
There are some differences from the design: the background color is darker, and there's no `box-shadow`. Is this intentional?